### DISCUSSION TITLE
Random location/scale individual estimators

### DISCUSSION THREAD

**Discussion by justinkirion** [reach-lab/MixWildGUI#163] [Help Wanted] [answered]
Hello !

I wonder if there is a way to access Empirical Bayes estimators of random location/scale values for each subject. It seems that MixWild computes them, since they can be used in stage 2 models, and I would like to know if it is possible to store them in a file.

Thank you in advance for your answer !

**Comment by adityaponnada** [comment, marked as answer]
Hello, I think it is saved in the work folder. This folder is in the same
location as your data set and should have the name MIXWILD*********** with
the time stamp. Let me know if you are able to find it

**Reply by justinkirion** [reply]
Thanks, I didn't see the work folder at first. It seems that the file data_ebvar.dat contains the individual results I am looking for. However, there is another file named data_ebvar2.dat, which returns something similar in shape as data_ebvar.dat, but with different results. Could you explain the difference between this two files ? 

EDIT : After examining more thoroughly, I think the file ebvar.dat returns the standardized parameters with mean 0 and variance 1, while the file ebvar2.dat returns the parameters with the variance defined in global model results, and taking into account covariance between effects. But I may be wrong with this interpretation.

**Reply by hedeker** [reply]
your are correct.  sorry for the lateness with this reply.
